details_land.md

```diff
@@ -32,23 +32,23 @@ Finally, because the logit sharing approach reflects non-linear representations
+Our approach employs a nesting strategy to capture the varying substitutability across different land categories. Figure 1 illustrates the nesting diagram of land within a subregion. The hierarchy begins with all land, which is then divided into two main nodes: agro-forestry land and non-agricultural land categories. The latter category can be further subdivided if necessary. The next layer includes two nodes: non-pasture land and pasture land. The pasture land node comprises two competing uses: managed pasture (grazed land for livestock) and unmanaged pasture. The agro-forestry (non-pasture) node consists of three competing nodes: shrub and grasslands, forest lands, and croplands. Shrublands and grasslands are treated separately as unmanaged land categories to control their substitutability. The forestland node competes with the total cropland node. Within forestland, managed and unmanaged forest leaves are considered, with the addition of woody biomass options in specific regions and scenarios. It is important to note that biomass crops are introduced into the model in 2025, as they are not included in the base data.
 
+Crop areas are further subdivided into irrigated/rainfed and high/low fertilizer usage categories (see Figure 2 for more details). This specification allows for adjusting the level of substitution across categories by varying the logit parameters. Figure 2 also indicates the corresponding files in the `gcamdata` system that process the inputs for each land type. For example, processing for purple leaves can be found in [zchunk_L2242.land_input_4_irr_mgmt.R](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/R/zchunk_L2242.land_input_4_irr_mgmt.R) and [zchunk_batch_land_input_4_IRR_MGMT_xml.R](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/R/zchunk_batch_land_input_4_IRR_MGMT_xml.R), with the final inputs set in a file named `land_input_4_IRR_MGMT.xml`). Note that only a subset of the GCAM crop commodities are shown. 
```

Comment:
`high/low fertilizer usage categories`:  we actually use the same fertilizer coefficients for all techs within a basin, right?  It's more like hi/low management (and therefore yield).  Can we rephrase?

Reply:
Also, the links will change because of the chunk rename... we should update accordingly.